// @nauverse/react-aurora-background/dist/cjs/lib/constants.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.BUBBLE_MAX_RANDOM_HEIGHT = exports.BUBBLE_MIN_RANDOM_HEIGHT = exports.BUBBLE_MAX_RANDOM_WIDTH = exports.BUBBLE_MIN_RANDOM_WIDTH = exports.DEFAULT_CONFIG = exports.BLUR_AMOUNT = exports.GRADIENT_COLORS = void 0;
const colors_1 = require("./colors");
exports.GRADIENT_COLORS = [
    colors_1.COLOR_RADICAL_RED,
    colors_1.COLOR_DODGER_BLUE_2,
    colors_1.COLOR_YELLOW,
    colors_1.COLOR_SHAMROCK,
];
exports.BLUR_AMOUNT = '10vw';
exports.DEFAULT_CONFIG = {
    colors: [
        colors_1.COLOR_RADICAL_RED,
        colors_1.COLOR_DODGER_BLUE_2,
        colors_1.COLOR_YELLOW,
        colors_1.COLOR_SHAMROCK,
    ],
    numBubbles: 4,
    animDuration: 20,
    blurAmount: exports.BLUR_AMOUNT,
    bgColor: colors_1.COLOR_DODGER_BLUE_2,
    useRandomness: false,
};
exports.BUBBLE_MIN_RANDOM_WIDTH = 48;
exports.BUBBLE_MAX_RANDOM_WIDTH = 52;
exports.BUBBLE_MIN_RANDOM_HEIGHT = 48;
exports.BUBBLE_MAX_RANDOM_HEIGHT = 52;
